Source/Core/Core/HW/SI/SI_DeviceGCController.cpp

```diff
@@ -318,15 +312,12 @@ void CSIDevice_GCController::SendCommand(u32 command, u8 poll)
       INFO_LOG_FMT(SERIALINTERFACE, "PAD {} set to mode {}", m_device_number, m_mode);
     }
   }
-  break;
-
-  default:
+  else if (controller_command.command != 0x00)
   {
+    // Costis sent 0x00 in some demos :)
```

Comment:
Seems to be a typo here

Reply:
I'm fairly certain their name is Costis, or didn you mean something else? I don't see any other potential typo.

Reply:
It seemed that they meant "Cost is" I don't know who Costis is but anyhow the meaning isn't clear.

Reply:
They made a few homebrew things like SDLoader/PSOLoader. Assuming that comment is about them.